.gitignore

```diff
@@ -21,3 +21,4 @@ gradlew
 gradlew.bat
 settings.gradle
 .idea/
+appsettings.json
```

Comment:
Based on my understanding, the client who pulls the SDK now, won't have the ability to run tests for the sandbox to see how the integration is supposed to work. Also, since we are not putting even structure of this file, then how the clients are supposed to run tests to verify that everything is fine and configured correctly on their end?

Reply:
Could we use https://mkyong.com/java/java-properties-file-examples/ properties file (market-standard for Java and simpler option)?